Dockerfile

```diff
@@ -95,9 +95,9 @@ SHELL ["/bin/bash", "-c"]
 
 # Install golang
 ARG BUILDARCH
-ADD https://golang.org/dl/go1.20.2.linux-${BUILDARCH}.tar.gz /tmp/go1.20.2.tar.gz
+ADD https://go.dev/dl/go1.21.1.linux-${BUILDARCH}.tar.gz /tmp/go1.21.1.tar.gz
```

Comment:
Technically, the `golang.org` URL should still work, right? If the `go.dev` URLs are preferred now, do we want to decouple the version update from the URL update?

Reply:
`golang.org` only exists now as a redirect, this shouldn't functionally change anything.